lib/platform/bitbucket/index.ts

```diff
@@ -130,10 +135,17 @@ export async function initRepo({
     throw err;
   }
 
+  const { hostname } = URL.parse(endpoint);
+
+  // Converts API hostnames to their respective HTTP git hosts:
+  // `api.bitbucket.org`  to `bitbucket.org`
+  // `api-staging.<host>` to `staging.<host>`
+  const hostnameWithoutApiPrefix = /api[.|-](.+)/.exec(hostname)[1];
```

Comment:
this throws reference error, if hostname does not start with api

Reply:
There's already a warn if using a custom endpoint anyway..